### ISSUE TITLE
Cannot find module 'hbs'

### ISSUE THREAD

**Issue by ChrisJefferson** [nikku/wuffle#175]
__Please Acknowledge the Following__

* [x ] I've followed the [setup and run instructions](https://github.com/nikku/wuffle/blob/main/docs/SETUP.md)

We cannot help you, otherwise.

__At which step in the setup are you stuck?__

I first ran `npm install waffle; npx waffle`, this gave me:

```
INFO (wuffle:run): Running Wuffle v0.57.0 in /home/caj/progs/wuffle
INFO (wuffle:run): Running first time setup
INFO (wuffle:run): Validating setup
ERROR (wuffle:run): No <url> configured in app.yml
ERROR (wuffle:run): Please correct above errors and restart
```

I might be mis-reading the setup, but I thought I would be guided through this, but I went and set the default values for `url` and `name` in `app.yml`, by uncommenting them (this might have been my first mistake).

I then ran again, but the website won't come up, and I get the following error:

```
INFO (wuffle:run): Running Wuffle v0.57.0 in /home/caj/progs/wuffle
INFO (wuffle:run): Running first time setup
INFO (wuffle:run): Validating setup
(node:24247) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
INFO (wuffle:run): Validating configuration
WARN (wuffle:run): Not running in production mode
WARN (wuffle:run): Missing env.APP_ID
WARN (wuffle:run): Missing env.BASE_URL
WARN (wuffle:run): Missing env.GITHUB_CLIENT_ID
WARN (wuffle:run): Missing env.GITHUB_CLIENT_SECRET
WARN (wuffle:run): Missing env.SESSION_SECRET
WARN (wuffle:run): Missing env.WEBHOOK_SECRET
INFO (server): Running Probot v12.3.3 (Node.js: v21.0.0)
INFO (server): Listening on http://localhost:3000
WARN (wuffle:run): Visit http://localhost:3000 to create a GitHub App that connects us to GitHub
INFO (http): GET / 302 - 2ms
INFO (http): GET / 302 - 3ms
ERROR (event): Cannot find module 'hbs'
Require stack:
- /home/caj/progs/wuffle/node_modules/express/lib/view.js
- /home/caj/progs/wuffle/node_modules/express/lib/application.js
- /home/caj/progs/wuffle/node_modules/express/lib/express.js
- /home/caj/progs/wuffle/node_modules/express/index.js
- /home/caj/progs/wuffle/node_modules/probot/lib/server/server.js
- /home/caj/progs/wuffle/node_modules/probot/lib/index.js
- /home/caj/progs/wuffle/node_modules/wuffle/lib/probot/CustomProbot.js
- /home/caj/progs/wuffle/node_modules/wuffle/lib/probot/index.js
- /home/caj/progs/wuffle/node_modules/wuffle/bin/run.js
- /home/caj/progs/wuffle/node_modules/wuffle/bin/wuffle
    Error: Cannot find module 'hbs'
    Require stack:
    - /home/caj/progs/wuffle/node_modules/express/lib/view.js
    - /home/caj/progs/wuffle/node_modules/express/lib/application.js
    - /home/caj/progs/wuffle/node_modules/express/lib/express.js
    - /home/caj/progs/wuffle/node_modules/express/index.js
    - /home/caj/progs/wuffle/node_modules/probot/lib/server/server.js
    - /home/caj/progs/wuffle/node_modules/probot/lib/index.js
    - /home/caj/progs/wuffle/node_modules/wuffle/lib/probot/CustomProbot.js
    - /home/caj/progs/wuffle/node_modules/wuffle/lib/probot/index.js
    - /home/caj/progs/wuffle/node_modules/wuffle/bin/run.js
    - /home/caj/progs/wuffle/node_modules/wuffle/bin/wuffle
        at Module._resolveFilename (node:internal/modules/cjs/loader:1138:15)
        at Module._load (node:internal/modules/cjs/loader:979:27)
        at Module.require (node:internal/modules/cjs/loader:1229:19)
        at require (node:internal/modules/helpers:177:18)
        at new View (/home/caj/progs/wuffle/node_modules/express/lib/view.js:81:14)
        at Function.render (/home/caj/progs/wuffle/node_modules/express/lib/application.js:587:12)
        at ServerResponse.render (/home/caj/progs/wuffle/node_modules/express/lib/response.js:1039:7)
        at /home/caj/progs/wuffle/node_modules/wuffle/lib/probot/apps/setup.js:41:11
        at Layer.handle [as handle_request] (/home/caj/progs/wuffle/node_modules/express/lib/router/layer.js:95:5)
        at next (/home/caj/progs/wuffle/node_modules/express/lib/router/route.js:144:13)
```

**Comment by nikku** [comment]
Thanks for reporting. 

I can confirm this is currently broken, and it is not your fault. You can go through the [manual installation steps](https://github.com/nikku/wuffle/blob/main/docs/SETUP.md#manual-github-app-setup). I'll see that I fix this as soon as possible.

**Comment by nikku** [comment]
Should be fixed with [v0.58.0](https://github.com/nikku/wuffle/releases/tag/v0.58.0).
